.babelrc

```diff
@@ -1,3 +1,12 @@
 {
-     "presets": ["@babel/preset-env"]
+     "presets": [
```

Comment:
how many line spacing are we adhering to?. I'll advice you setup a `.editorconfig` to help with this. Make sure the team uses that

Reply:
thanks will adhere to this and inform the team moving forward